### PR TITLE
transformed dropbox to a reusable lib

### PR DESCRIPTION
**Pull request by yashjain28**

### CODE REVIEW

**Comment by rreinold** [comment]
Assets missing the Code Library

**Comment by rreinold** [comment]
Need to move API below Usage, and remove Thank You

**Comment by rreinold** [comment]
We should have a better description in Overview

**Comment by rreinold** [comment]
remove '...' in assets

**Comment by rreinold** [comment]
Move ' ClearBlade.init' to outside this library, tell the user to run `ClearBlade.init` before using this library, then simplify the method signature for the Library

**Comment by yashjain28** [comment]
00d61323a22bc446f5e5dae1e1aec8ebd18736db

File: Readme.md [hunk 1]
```diff
@@ -20,38 +20,69 @@ This package contains an example S3 Object to fetch, and can be run upon a fresh
...
 
-### ...
 
-## Thank you
+## API
 
-Powered by ClearBlade Enterprise IoT Platform: [https://platform.clearblade.com](https://platform.clearblade.com)
+<a name="DropboxLib"></a>
+
+## DropboxLib : <code>Object</code>
```
Comment (rreinold) [inline Readme.md]: API should be Dropbox Object

File: jsdoc.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<a name="FetchDropboxAsset"></a>
```
Comment (yashjain28) [inline jsdoc.md:17]: I thought we just have jsdoc for services, just for developers reference and not to be put in the readme. We already have an example in Library jsdoc.

**Pull request merged**